src/components/icons/Icon.jsx

```diff
@@ -68,24 +68,52 @@ export const ICON_COLOR = {
 
 export const SIZE = [120, 118, 96, 94, 64, 62, 48, 46, 38, 32, 30, 26, 24, 22, 20, 18, 16, 14, 10, 8];
 
-const Icon = ({color, size = 24, type, className, ...props}) => {
+const Icon = ({color, size = 24, type, children, className, ...props}) => {
   const iconClass = classNames('sg-icon', {
     [`sg-icon--${color}`]: color,
     [`sg-icon--x${size}`]: size
   }, className);
   const iconType = `#icon-${type}`;
 
   return (
-    <svg {...props} className={iconClass}>
-      <use xlinkHref={iconType} />
-    </svg>
+    <div {...props} className={iconClass}>
+      {type ?
```

Comment:
Maybe an error would be nice, if both are provided - someone might spend hours debugging in the future